hack/deployer/runner/ocp.go

```diff
@@ -166,6 +171,8 @@ func (d *OcpDriver) Execute() error {
 }
 
 func (d *OcpDriver) auth() error {
+	_ = NewCommand(fmt.Sprintf("gcloud config set project %s", d.ctx["GCloudProject"])).Run()
```

Comment:
It is, but I can triple check if needed. Without it, the OCP deployment may fail since it'll try to use the default configured account /project. This was not set for me when I ran it.